frontends/web/src/components/headerssync/headerssync.tsx

```diff
@@ -14,44 +16,61 @@
  * limitations under the License.
  */
 
-import { Component, h } from 'preact';
-import { translate } from 'react-i18next';
+import { Component, h, RenderableProps } from 'preact';
 import { subscribe } from '../../decorators/subscribe';
+import { translate, TranslateProps } from '../../decorators/translate';
+import { CoinCode } from '../../routes/account/account';
+import Spinner from '../spinner/ascii';
 import * as style from './headerssync.css';
 
-@translate()
-// @ts-ignore (generics need to be typed explicitly once converted to TypeScript)
-@subscribe(props => ({ status: 'coins/' + props.coinCode + '/headers/status' }))
-export default class HeadersSync extends Component {
-    constructor(props) {
-        super(props);
-        this.state = { show: 0 };
+interface HeadersSyncProps {
```

Comment:
So, are the interface types prefixed with `I` or not. What's the new convention?

Reply:
sure, we had it only for the exported interfaces that are going to be shared everywhere, but fine by me if we start using it also for internal stuff.